.github/workflows/scripts/nugetSlackNotifications/packageInfo.json

```diff
@@ -0,0 +1,32 @@
+[
+    { "packageName": "amazon.lambda.apigatewayevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.applicationloadbalancerevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.cloudwatchevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.dynamodbevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.kinesisevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.kinesisfirehoseevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.s3events", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.simpleemailevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.snsevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "amazon.lambda.sqsevents", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "elasticsearch.net", "ignorePatch": false, "ignoreMinor": false },
+    { "packageName": "elastic.clients.elasticsearch", "ignorePatch": true, "ignoreMinor": false },
```

Comment:
Should we add an optional json field that allows us to explain why certain releases are ignored?

Reply:
Sure!